torch/csrc/jit/codegen/cuda/ir_base_nodes.cpp

```diff
@@ -29,6 +29,12 @@ Expr* Statement::asExpr() {
   return static_cast<Expr*>(this);
 }
 
+void Statement::print() {
+  IRPrinter ir_printer(std::cout);
+  ir_printer.handle(this);
+  std::cout << std::endl;
```

Comment:
Why not just `std::cout << this << std::endl;`? Do we need another way to print a stmt?

Reply:
std::cout << ... doesn't work nicely in the debugger. 

So I thought that it wouldn't hurt to have a consistent .print() method for all IR objects (we already have fusion.print()). What do you think?

Reply:
Sure, just wanted to know why.